pkg/k8s/client.go

```diff
@@ -27,6 +30,20 @@ import (
 	"github.com/openservicemesh/osm/pkg/service"
 )
 
+var (
```

Comment:
Can we move these to `pkg/errcode`  which has troubleshooting tooling built over it.